api/configuration/logging_controller.go

```diff
@@ -0,0 +1,56 @@
+package configuration
+
+import (
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/log"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/pkg/web"
+)
+
+// Controller logging configuration controller
```

Comment:
If this is only for logging, rename to `configuration.LoggingController`. Otherwise change the godoc